### PR TITLE
Feat jordi5107 blog

### PR DESCRIPTION
**Pull request by jordi5107**
Added a new blog post on Dev.to on [What is AppWrite and who i setup in my project?](https://dev.to/jordi5107/what-is-appwrite-and-who-i-setup-in-my-project-3af)

Sorry for merge problems.

### CODE REVIEW

**Review by TheLearneer — COMMENTED** [review]
There seems to be a fanart commit alongside the blog commit.
Also it seems you removed some existing links. 😓

**Comment by jordi5107** [comment]
Sorry for that i think i fixed

**Review by Meldiron — CHANGES_REQUESTED** [review]
Let's increase the length of the article to give it more value. There is soooo much to write about

**Comment by jordi5107** [comment]
I have improved the post and added more content to it, I hope it is better now 🙂.

**Review by Meldiron — CHANGES_REQUESTED** [review]
Let's fix all `AppWrite` to `Appwrite` and correctly mark code blocks so we get syntax highlighting (its markdown feature supported by Dev.to)

**Comment by jordi5107** [comment]
Fix that things.

**Comment by jordi5107** [comment]
Any feedback?

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Screenshot 2021-11-28 at 7 58 06 PM

Please fix the title of your blog to use `Appwrite` and not `AppWrite`. 

Please address the merge conflict as well

**Comment by jordi5107** [comment]
Fixed

File: README.md [hunk 1]
```diff
@@ -149,6 +149,8 @@ Submit your projects and win some Awesome Appwrite Swag 🤩
 * [Appwrite Storage Service](https://dev.to/yashp1210/appwrites-storage-service-ned)
 * [Appwrite + Zapier = Magic](https://dev.to/ruthlessruler/appwrite-zapier-magic-12gk)
 * [How To Generate and Store Map Previews in Ruby with Appwrite](https://dev.to/oliverpham/how-to-generate-and-store-map-previews-in-ruby-with-appwrite-43ha)
+* [What is AppWrite and who i setup in my project?](https://dev.to/jordi5107/what-is-appwrite-and-who-i-setup-in-my-project-3af)
```
Comment (christyjacob4) [inline README.md:152]:
```suggestion
* [What is Appwrite and who i setup in my project?](https://dev.to/jordi5107/what-is-appwrite-and-who-i-setup-in-my-project-3af)
```

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged :partying_face:

Please reach out to me in Discord to claim your hacktoberfest swag!

As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord so that we can add you as a contributor to our organization.

Once added, you can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite membership badge will show up on your personal GitHub profile.

**Pull request merged**